lib/battlestation/cli.rb

```diff
@@ -133,45 +133,24 @@ def install_packages(current_dirname)
         'ios-sim',
         'python',
         'rbenv',
-        'terminal-notifier',
-        'youtube-dl',
         'gh',
       ]
-      # Slowly bringing code into ruby from the monolithic setup script.
-      system 'bash', '-c', %{
-#######################################
-# Prints the provided message to STDOUT in green.
-#
-# Arguments:
-#   Info message
-#######################################
-echo_info() {
-  printf "$(tput setaf 2)%s$(tput sgr 0)\n" "$*" >&2;
-}
-
-#######################################
-# Installs the specified Homebrew package if it isn't installed. If it is, tries to upgrade the
-# package.
-#
-# Arguments:
-#   Name of package
-#######################################
-install_or_upgrade_package() {
-  if ! #{full_homebrew_path} list "$1" >/dev/null 2>&1; then
-    echo_info "Installing $1 via Homebrew"
-    #{full_homebrew_path} install "$1"
-  else
-    # TODO: find a way to only update if needed to prevent errors in console.
-    #{full_homebrew_path} upgrade "$1"
```

Comment:
Previously we would always upgrade which would result in a warning if there wasn't a newer version.